source/mainnet/net/mobile-wallet/update-baker-mw.rst

```diff
@@ -0,0 +1,132 @@
+.. _update-baker-mw:
+
+=====================================
+Change baker options in Mobile Wallet
+=====================================
+
+#. Go to **Accounts**. Tap on the balance area of the baker account you want update or tap **More** |moredetails|.
+
+#. In the hamburger menu |hamburger| tap **Baking**.
+
+#. In the Baker status screen tap **Update baker settings**.
+
+    .. image:: ../images/mobile-wallet/baker-status-mw.png
+
+You have four options:
+
+    - Update baker stake
+    - Update pool settings
+    - Update baker keys
+    - Stop baking
+
+.. image:: ../images/mobile-wallet/baker-hamburger-menu.png
+
+Update baker stake
+==================
+
+If you choose **Update baker stake** you see your balance and the current Baker stake amount. Enter the amount you want to stake. This is the full amount to stake and is not added to the existing stake. You can also choose to adjust your restake setting between **Yes, restake** rewards or **No, don’t restake** rewards. Tap **Continue**.
+
+.. image:: ../images/mobile-wallet/update-baker-stake-amt-mw.png
+
+On the overview screen, check the information. Once you are satisfied, tap **Submit baker transaction**.
+
+Once the transaction is submitted you see a confirmation screen. Tap **Finish** to complete the action.
```

Comment:
```suggestion
Once the transaction is submitted you will see a confirmation screen. Tap **Finish** to complete the action.
```

Reply:
Using future tense is a big no-no in technical documentation. I will reword the sentence to keep it in present tense without the word "see" in it.